.pipeline/scripts/get-release-versions.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
(Comment)

I'm sorry for all the python fans, but this could've been a single CLI line, like:
```
mvn help:evaluate -Dexpression=project.version -q -DforceStdout
```

But I'm okay with leaving the XML parsing, since this is what we do in the `set-release-version.py` anyway.